### ISSUE TITLE
Fix or update Apache.Avro nuget packages

### ISSUE THREAD

**Issue by valentynbulakh** [Azure/azure-functions-docker#612]
Hi
We are using azure-functions to build docker image,
After build we do Aqua Scanner to check Vulnerabilities.
The latest image 4.1.3 hat 4  Vulnerabilities related to Apache.Avro.
https://nvd.nist.gov/vuln/detail/CVE-2021-43045

This issue if fixed in Apache.Avro version 1.11.0.
In azure-functions image version is 1.10.2.

Can you update image to use latest Apache.Avro version or how we can do this?
Tried to update like this:
RUN python3 -m pip install avro 
RUN dotnet add package Apache.Avro --version 1.11.0
RUN dotnet tool install --global Apache.Avro.Tools --version 1.11.0

But has no luck, it's updating it, but Aqua Scanner still shows that old version is present.

Maybe image has some library or dependency that should be updated.

**Comment by v-bbalaiagar** [comment]
Hi @valentynbulakh , Thank you for your feedback! We would like to know what container image you are using.

**Comment by valentynbulakh** [comment]
@v-bbalaiagar 
Currently [dotnet:3.1.4](https://mcr.microsoft.com/azure-functions/dotnet:3.1.4).
The latest version that available version is 4.1.3, 
in this version Apache Avro updated only to 1.10.2 version.

Vulnerability issue fixed in 1.11.0 version.

**Comment by v-bbalaiagar** [comment]
Hi @pragnagopa , Could you please look into this issue.

**Comment by valentynbulakh** [comment]
Hi @pragnagopa, when you are planning to release new image?

**Comment by valentynbulakh** [comment]
also latest images: 
mcr.microsoft.com/azure-functions/dotnet:4
mcr.microsoft.com/dotnet/sdk:3.1-bullseye
have move vulnerabilities that oldest
Please see attached screenshots
![Capture](https://user-images.githubusercontent.com/98530651/154997212-bd47b97f-ef10-4b8f-a640-aa39df1d13ad.JPG)
![Capture1](https://user-images.githubusercontent.com/98530651/154997215-32d98de0-4d4c-4fe6-8163-8dd0316211cf.JPG)

**Comment by pragnagopa** [comment]
> Hi @pragnagopa, when are you planning to release new image?

Yes, we update images roughly about every two weeks - this will ensure updates in the dependency packages are pulled in. ETA for next update first week of March. If this is needed sooner  - please use Custom Container.

cc @CooperLink

**Comment by valentynbulakh** [comment]
@CooperLink, @pragnagopa
Hi.
Any updates when it will be fixed?

**Comment by pragnagopa** [comment]
@valentynbulakh - as mentioned earlier - `Apache.Avro` is not a direct dependency in Functions Images - whenever dependency updates to latest - Functions Images will include that.

**Comment by valentynbulakh** [comment]
@CooperLink, @pragnagopa 
We already using custom image
as base image we are using mcr.microsoft.com/dotnet/aspnet:3.1-alpine3.14
and as build mcr.microsoft.com/dotnet/core/sdk:3.1-buster 

Both images are has the similar issues.

**Comment by pragnagopa** [comment]
@valentynbulakh - thanks for the details - please open issue - https://github.com/dotnet/dotnet-docker  as mentioned earlier - the packages you are referring to are not direct dependencies for the images built in this repo

**Comment by valentynbulakh** [comment]
@pragnagopa Thanks

**Comment by mthalman** [comment]
> @valentynbulakh - thanks for the details - please open issue - https://github.com/dotnet/dotnet-docker as mentioned earlier - the packages you are referring to are not direct dependencies for the images built in this repo

@pragnagopa - I'm one of the maintainers of https://github.com/dotnet/dotnet-docker. We don't install anything from Apache.Avro in our images. I _do_ see files from Apache.Avro in the Azure Functions image though (see output below). So I would suggest investigating further on your side. This is clearly related to Azure Functions.

```console
> docker run --rm -it --entrypoint /bin/sh mcr.microsoft.com/azure-functions/dotnet:4
# find -name Avro*
./FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/2.9.1/bin_v3/linux-x64/Avro.dll
./FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/2.9.1/bin/Avro.dll
./FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/3.6.1/bin_v3/linux-x64/Avro.dll
./FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/3.6.1/bin/Avro.dll
```

**Comment by valentynbulakh** [comment]
@pragnagopa, @mthalman  Do you have any updates?
